hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/webapp/FederationInterceptorREST.java

```diff
@@ -958,7 +956,7 @@ public NodesInfo getNodes(String states) {
     try {
       subClustersActive = getActiveSubclusters();
     } catch (Exception e) {
-      LOG.error("Cannot get nodes: {}", e.getMessage());
+      LOG.error("Cannot get nodes: {}", e.getMessage(), e);
```

Comment:
Do we need the full stack trace?

Reply:
This part does not require full stack trace. I will refactor this part of the code to use `invokeConcurrent`.